up & going/ch3.md

```diff
@@ -39,19 +39,19 @@ The claims I make regarding delegation versus inheritance come not from a dislik
+El tercer título de esta serie primeramente se enfoca en abordar otro tema muy controversial: la coerción de tipos. Quizás ningún tópico causa mayor frustración en los desarrolladores de JS que cuando se habla acerca de las confusiones alrededor de la coerción implícita.
 
-By far, the conventional wisdom is that implicit coercion is a "bad part" of the language and should be avoided at all costs. In fact, some have gone so far as to call it a "flaw" in the design of the language. Indeed, there are tools whose entire job is to do nothing but scan your code and complain if you're doing anything even remotely like coercion.
+En gran medida, la sabiduría convencional es que la coerción implícita es una "mala parte" del lenguaje y debería evitarse a toda costa. De hecho, algunos han ido tan lejos en llamarlo una "falla" en el diseño del lenguaje. En realidad hay herramientas cuyo su único trabajo es no hacer nada más que escanear su código y quejarse si usted está haciendo algo remotamente parecido a la coerción.
 
-But is coercion really so confusing, so bad, so treacherous, that your code is doomed from the start if you use it?
+Pero es realmente la coerción tan confuso, tan malo, tan traicionero, que su código está condenado desde el momento que usted lo usa?
 
-I say no. After having built up an understanding of how types and values really work in Chapters 1-3, Chapter 4 takes on this debate and fully explains how coercion works, in all its nooks and crevices. We see just what parts of coercion really are surprising and what parts actually make complete sense if given the time to learn.
+Yo digo que no. Después de haber abordado y entendido cómo los tipos y valores realmente funcionan en los Capítulos 1-3, el Capítulo 4 se encarga de este debate y explica por completo cómo la coerción funciona en todos los aspectos. Veremos qué partes de la coerción son realmente sorprendentes y qué partes realmente tienen completo sentido si se les da el tiempo para aprender.
 
-But I'm not merely suggesting that coercion is sensible and learnable, I'm asserting that coercion is an incredibly useful and totally underestimated tool that *you should be using in your code.* I'm saying that coercion, when used properly, not only works, but makes your code better. All the naysayers and doubters will surely scoff at such a position, but I believe it's one of the main keys to upping your JS game.
+Pero no estoy sólo sugiriendo que la coerción es razonable y aplendible, estoy afirmando que la coerción es una herramienta increíblemente útil y totalmente subestimada que *usted debería usar en su código*. Estoy diciendo que esa coerción, cuando se usa adecuadamente, no sólo funciona, sino hace que su código sea mejor. Seguramente todos los que no están de acuerdo con esto se burlarán de esta posición sobre la coerción, pero creo que es una de las principales claves para aumentar tu juego en JS.
```

Comment:
We can use 'se puede aprender' instead of 'aprendible' that what you meant on first line

Reply:
Done @mballeng91 , thanks!